### PR TITLE
ss-152: [BUG] Wrong website layout on different screens

### PR DESCRIPTION
**Pull request by teslmik**
Screen width 1900px
Screenshot 2023-03-25 at 23-18-01 Vite React TS

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
Please add screenshots to PR, not to the issues

File: frontend/src/assets/css/variables/container-width-variables.scss
Comment (nikita-remeslov) [inline frontend/src/assets/css/variables/container-width-variables.scss:1]: can we make a global class to use container size?
```scss
.container {
    max-width: var(--container);
    width: 100%;
}
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/landing/components/footer/footer.tsx [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-        { to: AppRoute.UI, value: 'Blog' },
-        { to: AppRoute.UI, value: 'Contact' },
-    ];
+        { to: '#app', value: 'Mobile app' },
+        { to: '#about', value: 'About' },
+        { to: '#analytics', value: 'Analytics' },
+        { to: '#budget', value: 'Smart budget' },
+        { to: '#reviews', value: 'Reviews' },
+        { to: '#subscription', value: 'Subscription' },
+    ] as unknown as { to: ValueOf<typeof AppRoute>; value: string }[];
```
Comment (nikita-remeslov) [inline frontend/src/bundles/landing/components/footer/footer.tsx:19]: wrong type. It is not AppRoute

Comment (teslmik) [reply frontend/src/bundles/landing/components/footer/footer.tsx:19]: edit in another issue

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**